scripts/build_wasm.sh

```diff
@@ -68,16 +68,16 @@ build() {
             )
             ;;
         "samples")
-            NAMES=($(
+            NAMES=("$(
                 cargo metadata --no-deps --manifest-path "$CARGO_DIR/Cargo.toml" --format-version=1 |
                 jq '.packages | map(select(.targets[].kind | contains(["cdylib"]))) | map(.manifest_path | split("/")) | map(select(.[-3] == "samples")) | map(.[-2]) | .[]' -r
-            ))
-    esac
+            )")
+    ;; esac
 
     mkdir -p "$TARGET_DIR/$1"
-    for name in ${NAMES[@]}; do
+    for name in "${NAMES[@]}"; do
         out_file="$TARGET_DIR/$1/$name.wasm"
-        "${bin_kagami[@]}" wasm build "$CARGO_DIR/$1/$name" --profile=$PROFILE --out-file "$out_file"
+        "${bin_kagami[@]}" wasm build "$CARGO_DIR/$1/$name" --profile="$PROFILE" --out-file "$out_file" --cargo-args="--locked"
```

Comment:
I think `profile` can be moved to `--cargo-args` eventually.

Reply:
Created https://github.com/hyperledger-iroha/iroha/issues/5453